File: sum-of-digits/java/SumOfDigits.java
import java.util.Scanner;

public class SumOfDigits {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter a number: ");
        var input = scanner.nextInt();
        var sum = 0;

        while (input > 0) {
            sum = sum + input % 10;
            input = input / 10;
        }

        System.out.println("Sum of digits: " + sum);

        scanner.close();
    }
}

File: sum-of-digits/python/SumOfDigits.py
def main():
    number = int(input('Enter a number: '))
    sum = 0

    while number > 0:
        sum = sum + number % 10
        number = number // 10 # Note: // is an integer division operator

    print('Sum of digits:', sum)

if __name__ == '__main__':
    main()